docs/topics/funding.md

```diff
@@ -115,36 +111,50 @@ REST framework continues to be open-source and permissively licensed, but we fir
 
 ---
 
-## Making the business case
+## What funding has enabled so far
 
-Our [successful Kickstarter campaign](https://www.kickstarter.com/projects/tomchristie/django-rest-framework-3) demonstrates the impressive cost-reward ratio of shared funding of open-source software.
+* The [3.4](http://www.django-rest-framework.org/topics/3.4-announcement/) and [3.5](http://www.django-rest-framework.org/topics/3.5-announcement/) releases, including schema generation for both Swagger and RAML, a Python client library, a Command Line client, and addressing of a large number of outstanding issues.
+* The [3.6](http://www.django-rest-framework.org/topics/3.6-announcement/) release, including  JavaScript client library, and API documentation, complete with auto-generated code samples.
+* Tom Christie, the creator of Django REST framework, working on the project full-time.
+* Over 320 closed issues on [GitHub](https://github.com/tomchristie/django-rest-framework) since Tom Christie started working on the project full-time (June 2016 - March 2017).
```

Comment:
I got that wrong. The number seemed too low to me too. Will fix it. Thank you, Tom :)